### ISSUE TITLE
Deploying Graphql

### ISSUE THREAD

**Issue by And3y** [sanity-io/sanity-plugin-mux-input#36]
When I run the command `sanity graphql deploy` I receive this error:

```
TypeError: Cannot read property 'get' of undefined
    at o (~/Path-to-projct/sanity/node_modules/media-chrome/dist/index.js:82:259)
    at ~/Path-to-projct/sanity/node_modules/media-chrome/dist/index.js:82:1574
    at Object.<anonymous> (~/Path-to-projct/sanity/node_modules/media-chrome/dist/index.js:792:1271)
```

After downgrading to version `0.3.4` I was able to deploy again without any problems.

**Comment by dylanjha** [comment]
@And3y thanks for surfacing, we'll take a look at this!

**Comment by dylanjha** [comment]
Found it! with @KimPaow 😅 

`media-chrome` will try to detect server-side environments and shim out the `window` functionality that it relies upon in order to prevents things like this. It does so by using this [`windowShim`](https://github.com/muxinc/media-chrome/blob/main/src/js/utils/server-safe-globals.js)

The problem is here

```
export const isServer = !!(typeof window === 'undefined')
export const Window = isServer ? windowShim : window;
```

It turns out, in this sanity environment where the `sanity graphql deploy` command gets run `window` is not undefined. There already is a `window` defined, however that `window` does not have `customElements`. That's why it leads to this error of `Cannot read property 'get' of undefined`

I think the solution here might be for media-chrome to do a more robust check for `isServer` instead of checking `!!(typeof window === 'undefined)` we could do something more like:

```
export const isServer = !!(typeof window === 'undefined' || typeof window.customElements === 'undefined')
```

**Comment by judofyr** [comment]
This has now been fixed in media-chrome v0.1.1, but we're depending on v0.0.15. I'm guessing we'll have to upgrade the dependency here as well?

**Comment by KimPaow** [comment]
@judofyr Yes, we're working on this and you should see a new release very soon.
